test/unittest/tests/block/test_block_other.cpp

```diff
@@ -0,0 +1,125 @@
+/*-
+ * Copyright (c) 2014-present MongoDB, Inc.
+ * Copyright (c) 2008-2014 WiredTiger, Inc.
+ *	All rights reserved.
+ *
+ * See the file LICENSE for redistribution information.
+ */
+
+/*
+ * [block_other]: block.h
+ * This file unit tests the __wt_block_header_byteswap_copy and __wt_block_eligible_for_sweep
+ * functions.
+ */
+
+#include "wt_internal.h"
+#include <catch2/catch.hpp>
+
+static void
+test_block_header_byteswap_copy(WT_BLOCK_HEADER *from, WT_BLOCK_HEADER *to)
+{
+    WT_BLOCK_HEADER prev_from, prev_to;
+
+    // Save the original values before any potential byte re-orderings.
+    prev_from.disk_size = from->disk_size;
+    prev_from.checksum = from->checksum;
+    prev_to.disk_size = to->disk_size;
+    prev_to.checksum = to->checksum;
+
+#ifdef WORDS_BIGENDIAN
+    __wt_block_header_byteswap_copy(from, to);
+    REQUIRE(to->checksum == __wt_bswap32(prev_from.checksum));
+    REQUIRE(to->disk_size == __wt_bswap32(prev_from.disk_size));
+#else
+    // Test that the byte orderings remain the same in both block headers.
+    REQUIRE(from->disk_size == prev_from.disk_size);
+    REQUIRE(from->checksum == prev_from.checksum);
+    REQUIRE(to->disk_size == prev_to.disk_size);
+    REQUIRE(to->checksum == prev_to.checksum);
+#endif
+}
+
+TEST_CASE("Block header byteswap copy", "[block_other]")
+{
+    WT_BLOCK_HEADER from, to;
+
+    SECTION("Test case 1")
```

Comment:
Could we add some more descriptive for the section descriptions?

Reply:
I second Jie's comment - it's worth having a (brief) description of the test in SECTION labels.

Reply:
But there's no need to re-review after updating those labels.